tools/topology/sof-tgl-sdw-max98373-rt5682.m4

```diff
@@ -57,6 +59,13 @@ ifelse(PLATFORM, `adl',
 `	define(`SMART_ALH_INDEX', 0x202)
 	define(`SMART_ALH_PLAYBACK_NAME', `SDW2-Playback')
 	define(`SMART_ALH_CAPTURE_NAME', `SDW2-Capture')
+	# Add BT audio offload support
+	define(`BT_PIPELINE_PB_ID', `14') dnl DMIC_PIPELINE_KWD_ID + 1
+	define(`BT_PIPELINE_CP_ID', `15') dnl DMIC_PIPELINE_KWD_ID + 2
+	define(`BT_DAI_LINK_ID', `10')
+	define(`BT_PCM_ID', `9') dnl added after HDMI
```

Comment:
This is confusing, is this a pipeline or PCM ID?

Reply:
BT_PIPELINE_ is pipeline, BT_PCM_ is pcm ID, BT_DAI_LINK is BE id, sorry for the confusion, try to follow the same convention as DMIC example:

```
define(DMIC_PCM_48k_ID, `10')
define(DMIC_PIPELINE_48k_ID, `11')
define(DMIC_DAI_LINK_48k_ID, `4')
```

Reply:
So @yongzhi1 if I'm parsing Pierre's feedback correctly, this should be:
```
define(`BT_PCM_ID', `14')
```